junit/kube-api-test/client-inject/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kube-api-test-parent</artifactId>
+      <version>6.11-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kube-api-test-client-inject</artifactId>
+    <packaging>jar</packaging>
+    <name>Kube API Test - Client Injecting Support</name>
+
+    <dependencies>
+        <dependency>
+          <groupId>io.fabric8</groupId>
+          <artifactId>kube-api-test</artifactId>
+          <version>${project.version}</version>
```

Comment:
Should not be needed.

Define the dependency in the dependencyManagement section of the main pom.xml

This will also help in generating the BOM